README.md

```diff
@@ -54,6 +54,16 @@ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 
 `<zone>` should be replaced with a valid Azure availability zone like `us-east-1a`.
 
+`<image>` should be a WindowsServer image offering that defines the 2019-Datacenter-with-Containers SKU with version
+          17763.1457.2009030514 or later.Run the following command to list Azure image info
```

Comment:
Not sure if it should be `later` or `earlier`? 
Should this be later ?

Reply:
Earlier

Reply:
Thanks for catching this @PratikMahajan . This needs to be changed in openshift [docs](https://docs.openshift.com/container-platform/4.6/windows_containers/windows-containers-release-notes.html) as well